$MirOS: ports/print/cups/patches/patch-cgi-bin_template_c,v 1.2 2006/06/21 19:25:33 bsiegert Exp $
--- cgi-bin/template.c.orig	Fri May 19 19:39:05 2006
+++ cgi-bin/template.c	Tue Jul  4 17:55:16 2006
@@ -324,9 +324,9 @@ cgi_copy(FILE *out,			/* I - Output file
 	*/
 
         if (name[1])
-          sprintf(outval, "%d", cgiGetSize(name + 1));
+          snprintf(outval, sizeof(outval),"%d", cgiGetSize(name + 1));
 	else
-	  sprintf(outval, "%d", element + 1);
+	  snprintf(outval, sizeof(outval), "%d", element + 1);
 
         outptr = outval;
       }
